src/__snapshots__/App.test.js.snap

```diff
@@ -1,7 +1,402 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Snapshot test App 1`] = `
-<h1>
-  Online Book
```

Comment:
I think that we don't have to test this file while we are testing each component separately

Reply:
Okay